### PR TITLE
Fix read receipts?

### PR DESCRIPTION
**Pull request by SimonBrandner**
This should hopefully fix [#4377](https://github.com/vector-im/element-web/issues/4377). 

The thing is I am not certain... There has already been a [PR](https://github.com/matrix-org/matrix-react-sdk/pull/3892) which has attempted to do the same thing. After that PR was merged an [issue](https://github.com/vector-im/element-web/issues/12660) saying the RR is below the "Today" indicator appeared. Due to this, the PR was reverted. The issue was later closed - attempts at reproducing it were unsuccessful. 

So I decided to try to solve this myself. The thing is I ended up with a very similar code to the previous PR. I've attempted to reproduce the issue that caused the reversion unsuccessfully. That makes me think the issue might have been somewhere else and was fixed.

I've attempted to reproduce this in Matrix HQ and in my own testing room but everything seems fine.

Screenshot_20210124_091111

I can definitely put this to more tests if someone gave me directions on what to focus

### CODE REVIEW

File: res/css/views/rooms/_EventTile.scss [hunk 1]
```diff
@@ -258,17 +258,11 @@ $left-gutter: 64px;
     display: inline-block;
     width: 14px;
     height: 14px;
-    top: 29px;
+    top: -20px;
```
Comment (turt2live) [inline res/css/views/rooms/_EventTile.scss:261]: can we get a comment for where -20 comes from?

**Review by turt2live — APPROVED** [review]
otherwise looks plausible - thank you!

File: res/css/views/rooms/_EventTile.scss (hunk 1 above)
Comment (SimonBrandner) [reply res/css/views/rooms/_EventTile.scss]: Thanks for asking, made me realize that using pixels may not be a good idea...

**Review by turt2live — APPROVED** [review]
thanks! I'll merge this after we cut the RC just because it's a slightly risky change.

**Comment by turt2live** [comment]
right, let's try this :D

**Pull request merged**